crosshair/run_functional_tests.py

```diff
@@ -187,8 +188,13 @@ def strip_cwd(path: pathlib.Path) -> pathlib.Path:
                 # with a place holder to make these tests machine agnostic.
                 ##
 
-                stdout = stdout.replace(str(examples_dir), "<examples dir>")
-                stderr = stderr.replace(str(examples_dir), "<examples dir>")
+                path_re = re.compile(r'^.*[/\\]([_\w]+\.py):')
+
+                stdout = path_re.sub(
+                    r'<path prefix>/\1:', stdout.replace(str(examples_dir), "<examples dir>"))
```

Comment:
And then is the `<examples dir>` replacement not needed?
```suggestion
                    r'<path prefix>/\1:', stdout)
```